README.md

```diff
@@ -271,6 +272,38 @@ PrivateChannel channel = pusher.subscribePrivate("private-channel",
     });
 ```
 
+### Private encrypted channels
+
+Similar to Private channels, you can also subscribe to a
+[private encrypted channel](https://pusher.com/docs/channels/using_channels/encrypted-channels)
+which means the data passed through Pusher is encrypted and only possible to
+decrypt using the shared secret you have.
```

Comment:
I'm not quite sure about the final part of this sentence - whether it conveys the capability of the encryption clearly.

For reference, see the wording on the node server library, first sentence here: https://github.com/pusher/pusher-http-node#end-to-end-encryption-beta.

Reply:
Yeah, I struggled with the correct terminology to use - I've updated it to read:

```
Similar to Private channels, you can also subscribe to a
[private encrypted channel](https://pusher.com/docs/channels/using_channels/encrypted-channels).
Pusher fully supports end-to-end encryption of these private encrypted channels.
which means the data passed through Pusher is encrypted and only possible to
decrypt using the shared secret you have. This means that only you and your 
connected clients will be able to read your messages. Pusher cannot decrypt them.
```

Does that make it clearer?